pkg/rbac/tokenReview_test.go

```
@@ -94,6 +96,68 @@ func Test_IsValidToken_expiredCache(t *testing.T) {
+		},
+	}
+	mock_cache.tokenReviews[freshHash] = &tokenReviewCache{
+		authClient: mock_cache.authnClient,
+		meta:       cacheMetadata{updatedAt: time.Now()},
+		tokenReview: &authv1.TokenReview{
+			Status: authv1.TokenReviewStatus{Authenticated: true},
+		},
+	}
+
+	// Trigger a GetTokenReview call for any token — this runs eviction.
+	mock_cache.GetTokenReview(context.TODO(), "trigger-eviction")
```

golangci-lint (2.12.2)

[error] Error return value of `mock_cache.GetTokenReview` is not checked

(errcheck)